tokenizer/gpt2tok.cc

```diff
@@ -20,7 +20,7 @@
 #include "nlohmann/json.hpp"
 #include "kernels/kernels.h"
 #include "kernels/string_common.h"
-#include "unicode.h"
+#include "utils/unicode.h"

```

Comment:
this unicode source wasn't so common, and only be used in gpt2tokenizer.
Let's keep it where it was. We can move it until someday it will be reused by some other ops.

Reply:
Sure.